client/src/ui/css/ResultsDisplay.css

```diff
@@ -420,3 +420,15 @@ ul {
   padding-right: 8%;
   padding-left: 1.5%;
 }
+
+/* Not Found styling */
+ 
+.not-found-padding-top{
+  padding-top: 90px;
+}
+.not-found-text {
+  font-family: "Source Sans Pro", sans-serif;
+  font-size: 50px;
+  font-weight: var(--regular-weight);
+  text-align: center;
+}
```

Comment:
we should use [CSS Modules](https://github.com/css-modules/css-modules):
1. create separate file under: `css/NotFound.module.css`
2. move styles there (might have to rename)
3. import in `NotFound` component file
```jsx
import styles from "./css/NotFound.module.css"
```
4. use it by writing (for example) `<div className={styles.notFoundClassName}> ...` 

You can see a good example of this in Michelle's PR: https://github.com/cornell-dti/course-reviews-react-2.0/pull/293/files

Reply:
Thank you for your feedbacks! I made changes.